cmd/root.go

```diff
@@ -128,6 +132,30 @@ func setupLogging(baseDir string) (logging.Logger, error) {
 	return log, nil
 }
 
+// initConfig reads in config file and ENV variables if set.
+func initConfig() {
+	if cfgFile != "" {
+		// Use config file from the flag.
+		viper.SetConfigFile(cfgFile)
+	} else {
+		// Search for default config.
+		home, err := os.UserHomeDir()
+		cobra.CheckErr(err)
+		viper.AddConfigPath(home)
+		viper.SetConfigType("json")
+		viper.SetConfigName(".avalanche-cli")
```

Comment:
probably can just also use viper.SetConfigFile with the default config path

Reply:
moved to constants